code/src/components/MovieList.js

```diff
@@ -0,0 +1,23 @@
+import React, { useEffect, useState } from 'react'
+
+import { URL } from '../reusables/urls'
+
+import MovieThumb from './MovieThumb.js'
+
+const MovieList = ({ movieListType }) => {
+    const [movies, setMovies] = useState([]);
+    useEffect(() => {
+        fetch(URL(movieListType))
+            .then(response => response.json())
+            .then(receivedMovies => setMovies(receivedMovies.results))
+    }, [movieListType])
+    console.log(movies)
```

Comment:
Please remember to remove all console outputs from your code before submitting. 👮🏻